src/components/SegmentGroupControls.vue

```diff
@@ -110,22 +124,66 @@ function createSegmentGroup() {
 
   startEditing(id);
 }
+
+function createSegmentGroupFromImage(selectedImageID: string) {
+  if (!selectedImageID)
+    throw new Error('Cannot create a labelmap without a base image');
+
+  if (primarySelection.value) {
+    segmentGroupStore.convertImageToLabelmap(
+      { type: 'image', dataID: selectedImageID },
+      primarySelection.value
+    );
+  }
+}
 </script>
 
 <template>
   <div class="my-2" v-if="currentImageID">
     <div
       class="text-grey text-subtitle-2 d-flex flex-row align-center justify-space-between mb-2"
     >
-      <div>Segment Groups</div>
-      <v-btn
-        variant="tonal"
-        color="secondary"
-        density="compact"
-        @click.stop="createSegmentGroup"
-      >
-        <v-icon class="mr-1">mdi-plus</v-icon> New Group
-      </v-btn>
+      <v-container>
+        <v-row class="mb-1">
+          <div>Segment Groups</div>
+          <v-spacer />
+          <v-btn
+            variant="tonal"
+            color="secondary"
+            density="compact"
+            @click.stop="createSegmentGroup"
+          >
+            <v-icon class="mr-1">mdi-plus</v-icon> New Group
+          </v-btn>
+        </v-row>
+        <v-row>
+          <v-spacer />
+          <v-menu location="bottom">
+            <template v-slot:activator="{ props }">
+              <v-btn
+                variant="tonal"
+                color="secondary"
+                density="compact"
+                v-bind="props"
+              >
+                <v-icon class="mr-1">mdi-chevron-down</v-icon>From Image
+              </v-btn>
+            </template>
+            <v-list>
+              <v-list-item
+                v-for="(item, index) in nonDICOMImages"
+                :key="index"
+                @click="createSegmentGroupFromImage(item.id)"
+              >
+                {{ item.name }}
+                <v-tooltip activator="parent" location="end" max-width="200px">
+                  Convert to segment group
+                </v-tooltip>
+              </v-list-item>
+            </v-list>
+          </v-menu>
+        </v-row>
+      </v-container>
```

Comment:
What do you think of this layout:
![image](https://github.com/Kitware/VolView/assets/16823231/0bc88c62-a4dd-4624-9ffb-84e988e70fb8)

```
    <div
      class="text-grey text-subtitle-2 d-flex align-center justify-space-evenly mb-2"
    >
      <v-btn
        variant="tonal"
        color="secondary"
        density="compact"
        @click.stop="createSegmentGroup"
      >
        <v-icon class="mr-1">mdi-plus</v-icon> New Group
      </v-btn>
      <v-menu location="bottom">
        <template v-slot:activator="{ props }">
          <v-btn
            variant="tonal"
            color="secondary"
            density="compact"
            v-bind="props"
          >
            <v-icon class="mr-1">mdi-chevron-down</v-icon>From Image
          </v-btn>
        </template>
        <v-list>
          <v-list-item
            v-for="(item, index) in nonDICOMImages"
            :key="index"
            @click="createSegmentGroupFromImage(item.id)"
          >
            {{ item.name }}
            <v-tooltip activator="parent" location="end" max-width="200px">
              Convert to segment group
            </v-tooltip>
          </v-list-item>
        </v-list>
      </v-menu>
    </div>
```

Reply:
yea this looks better